lib/index.js

```diff
@@ -167,14 +167,21 @@ internals.root = function () {
         }
     };
 
-    root.assert = function (value, schema, message) {
+    root.assert = function (...args) {
 
-        this.attempt(value, schema, message);
+        this.attempt(...args);
     };
 
-    root.attempt = function (value, schema, message) {
+    root.attempt = function (value, schema, ...args/* [message], [options]*/) {
 
-        const result = this.validate(value, schema);
+        const first = args[0];
+        const message = (
```

Comment:
Wouldn't it be easier to just check if `args[0]` is an `object`?

Like:
```js
const firstArgAsOptions = typeof args[0] === 'object';
const message = firstArgAsOptions ? null : args[0];
const options = firstArgAsOptions ? args[0] : args[1];
```

Reply:
One of the possible values of `message` is a new Error instance, as `new Error()`.

Unfortunately, that has the type `'object'`
```js
> typeof (new Error())
'object'
```

Reply:
Oh, I missed that case, you're right!